docs/data-access.md

```diff
@@ -4,4 +4,32 @@ title: Applying for Access to Controlled Data
+1. Once you save your password, you will be taken to your ICGC Dashboard. From your Dashboard, click the link to start a new access application form from the box entitled “Your DACO Access Forms”.
+1. Complete the entire application form. All sections, as well as Appendices, are integral components of the application.
+1. **IMPORTANT: For the applicant and any authorized personnel, you will be asked to provide an “OpenID”, which means providing a valid google email address (Gmail or Gdrive) for each authorized personnel.** This will be the email address you will use to log in to the ARGO Data Platform for accessing controlled data.
+1. Once the application is completed you can submit it by “finalizing your DACO Application”. This action will prevent you from making further changes to your application without authorization from an administrator. 
+1. You will receive an email with further instructions. The next steps will also be listed on the confirmation page. Download and print your finalized application to add the necessary signatures. Once signed, you can fax or mail the application to the DACO office (the contact details are provided on the confirmation page and email).
+1. The DACO office will notify you when your application has been approved, or if they require any further information.
+1. Once your DACO access application is approved, your “OpenID” google email address will become associated with an ICGC ARGO account and you will be granted access to the controlled datasets available through the ARGO Data Platform. 
+
+## The DACO Approval Process
+
+The DACO approval process takes approximately 24-48 hours. 
+
+Your research project will be checked for conformity with the goals and [policies of ICGC ARGO](https://www.icgc-argo.org/page/76/e2-data-management-and-access) including, but not limited to, policies concerning the purpose and relevance of the research, the protection of the participants and the security of the participants' data. 
+
+The DACO office will notify you by email when your DACO application has been approved, or if they require any further information. Upon approval, access to controlled data will be granted for a **one year period** (starting from the date of acceptance). If your application is refused it remains possible to re-apply in the future.
+
+## Next steps after DACO Approval
+
+Please log in to the **[ARGO Data Platform](https://platform.icgc-argo.org/)** using the **same google email address (OpenID)** that is associated with your DACO account. 
+
+If you have been granted DACO access, you should see a green "DACO Approved" check mark on your [user profile page](https://platform.icgc-argo.org/user). This is also where you can access your API Token to use with the ICGC ARGO download client. For more information, check out [How to Download Data](data-download). 
```

Comment:
should we include a screenshot here?

Reply:
Yep! I'll include one in the next round of changes